internal/task/task/common/create_topology.go

```diff
@@ -86,7 +86,7 @@ func NewCreateTopologyTask(curveadm *cli.CurveAdm, dc *configure.DeployConfig) (
 	})
 	t.AddStep(&step.ContainerExec{ // create topology
 		ContainerId:  containerId,
-		Command:      fmt.Sprintf("%s build-topology", toolsBinary),
+		Command:      fmt.Sprintf("%s create-topology", toolsBinary),
```

Comment:
Is it will cause incompatibility?

Reply:
> Is it will cause incompatibility?

in this (pr)[https://github.com/opencurve/curve/pull/892],I will rename this command to create-topology